### PR TITLE
Add milliseconds to SLO IssueInstant

### PR DESCRIPTION
**Pull request by d60402**
SAML SLO is not working with SAP since IssueInstant does not include milliseconds as it is required by SAP IdP.

This changes the date formatting to use millisecond precision everywhere.

### CODE REVIEW

**Comment by sandrinodimattia** [comment]
@d60402 can you add a test to this? We need to make sure any change we make increases coverage (instead of decreasing it).

File: test/samlp.logout.session_store.tests.js [hunk 1]
```diff
@@ -44,6 +45,14 @@ describe('samlp logout with Session Participants - Session Provider', function (
   var sessions = [], returnError;
   var samlIdPIssuer = 'urn:fixture-test';
   
+  var frozenTime;
+  before(() => {
+    frozenTime = new Date(Date.now());
+    timekeeper.freeze(frozenTime)
```
Comment (luuuis) [inline test/samlp.logout.session_store.tests.js]: I think you can pass in a `number` directly, e.g. `timekeeper.freeze(Date.now())`. That avoids the `new Date().getTime()` calls below.

File: test/samlp.logout.custom_store.tests.js [hunk 2]
```diff
@@ -241,6 +250,7 @@ describe('samlp logout with Session Participants - Custom Provider', function ()
       it('should validate LogoutRequest to Session Participant', function () {
         expect(sessionParticipantLogoutRequest).to.exist;
         expect(xmlhelper.getIssueInstant(sessionParticipantLogoutRequest)).to.exist;
+        expect(new Date(xmlhelper.getIssueInstant(sessionParticipantLogoutRequest)).getTime()).to.equal(frozenTime.getTime());
```
Comment (luuuis) [inline test/samlp.logout.custom_store.tests.js]: Minor but I guess this assertion (once you remove the `new Date` part) makes the `.to.exist` assertion redundant

File: .travis.yml [hunk 3]
```diff
@@ -4,4 +4,4 @@ node_js:
   - 4.8.5
   - 6.9.2
 before_install:
-  - npm i -g npm@latest
\ No newline at end of file
+  - npm i -g npm@5
```
Comment (luuuis) [inline .travis.yml]: Why was this necessary?

Comment (luuuis) [reply .travis.yml]: This was for Node 4 compatibility.

**Comment by luuuis** [comment]
Fixes applied @ziluvatar , apologies for my bad copy-pasta.
